src/charts/stacked-bar.js

```diff
@@ -0,0 +1,1080 @@
+                             memo[v.stack] = colorScale(v.stack)
+                             memo[v.stack + item] = colorScale(v.stack)
+                        }
+                    })
+                    return memo;
+                }, {});
+        }
+
+        /**
+         * @param  {HTMLElement} container DOM element that will work as the container of the graph
+         * @private
+         */
+        function buildSVG(container) {
+            if (!svg) {
+                svg = d3Selection.select(container)
+                    .append('svg')
+                    .classed('britechart stacked-bar', true);
+
+                buildContainerGroups();
+            }
+
+            svg
+                .attr('width', width)
+                .attr('height', height);
+        }
+
+        /**
+         * Parses dates and values into JS Date objects and numbers
+         * @param  {obj} data Raw data from JSON file
+         * @return {obj}      Parsed data with values and dates
+         */
+        function cleanData(data) {
+            // could be rewritten using spread operator
+            /*
+                return data.map((d) => {...d, date: parseUTC(d[dateLabel], [valueLabel] : +d[valueLabel]})
+             */
+
+            return data.map((d) => {
+                // d.date = new Date(d[dateLabel]);
+                d.value = +d[valueLabel];
+
+                return d;
+            });
+        }
+
+        /**
+         * Draws the x and y axis on the svg object within their
+         * respective groups
+         * @private
+         */
+        function drawAxis(){
+            svg.select('.x-axis-group .axis.x')
+                .attr('transform', `translate( 0, ${chartHeight} )`)
+                .call(xAxis);
+
+            // if (forceAxisSettings !== 'custom') {
+            //     svg.select('.x-axis-group .month-axis')
+            //         .attr('transform', `translate(0, ${(chartHeight + monthAxisPadding)})`)
+            //         .call(xMonthAxis);
+            // }
+
+            svg.select('.y-axis-group.axis')
+                .attr('transform', `translate( ${-xAxisPadding.left}, 0)`)
+                .call(yAxis)
+                .call(adjustYTickLabels);
+
+            // Moving the YAxis tick labels to the right side
+            // d3Selection.selectAll('.y-axis-group .tick text')
+            //     .attr('transform', `translate( ${-chartWidth - yTickTextXOffset}, ${yTickTextYOffset})` );
+        }
+
+        /**
+         * Adjusts the position of the y axis' ticks
+         * @param  {D3Selection} selection Y axis group
+         * @return void
+         */
+        function adjustYTickLabels(selection) {
+            selection.selectAll('.tick text')
+                .attr('transform', `translate(${yTickTextXOffset}, ${yTickTextYOffset})`);
+        }
+
+         /**
+         * Draws grid lines on the background of the chart
+         * @return void
+         */
+        function drawGridLines(){
```

Comment:
On the line chart and stacked area chart we introduced a 'grid' configuration that would stablish if the chart has grids or not (horizontal, vertical, full)
It would be awesome if we could add that here too.